File: Tier2/src/main/java/via/sep3/group11/tier2/shared/DTOs/AgreementDTO.java
package via.sep3.group11.tier2.shared.DTOs;

import via.sep3.group11.tier2.shared.domain.Agreement;

public class AgreementDTO {
    private Agreement agreement;
    private String errorMessage;

    public AgreementDTO(Agreement agreement, String errorMessage) {
        this.agreement = agreement;
        this.errorMessage = errorMessage;
    }

    public Agreement getAgreement() {
        return agreement;
    }

    public String getErrorMessage() {
        return errorMessage;
    }


}

File: Program/Tier1/Shared/DTOs/AgreementDTO.cs
using Shared.Domain;

namespace Shared.DTOs;

public class AgreementDTO
{
    public Agreement Agreement { get; }
    public string ErrorMessage{ get; }

    public AgreementDTO(Agreement agreement, string errorMessage)
    {
        Agreement = agreement;
        ErrorMessage = errorMessage;
    }
}